src/App/Header/index.tsx

```diff
@@ -20,6 +20,7 @@ export default function Header({ links }: Props) {
           <div className="flex gap-2 md:gap-4 justify-self-end items-center">
             {!(
               location.pathname === appRoutes.signin ||
```

Comment:
```suggestion
              location.pathname === appRoutes.login ||
```
Nit: didn't we settle on login & signup?

Reply:
Will change this as part of Login rework https://linear.app/better-giving/issue/BG-1158/login-page